tools/cldr-apps/src/main/webapp/js/esm/cldrVettingParticipation.js

```diff
@@ -0,0 +1,304 @@
+import * as cldrLoad from "./cldrLoad.js";
+import * as cldrRetry from "./cldrRetry.js";
+import * as cldrStatus from "./cldrStatus.js";
+import * as cldrSurvey from "./cldrSurvey.js";
+import * as cldrText from "./cldrText.js";
+
+let nf = null; // Intl.NumberFormat initialized later
+
+/**
+ * Fetch the Vetting Participation data from the server, and "load" it
+ *
+ * Called as special.load
+ */
+function load() {
+  cldrInfo.showMessage(cldrText.get("vetting_participationGuidance"));
+
+  const url = getAjaxUrl();
+  const xhrArgs = {
+    url: url,
+    handleAs: "json",
+    load: loadHandler,
+    error: errorHandler,
+  };
+  cldrAjax.sendXhr(xhrArgs);
+}
+
+function loadHandler(json) {
+  if (json.err) {
+    cldrRetry.handleDisconnect(
+      json.err,
+      json,
+      "",
+      "Loading vetting participation data"
+    );
+    return;
+  }
+  const ourDiv = document.createElement("div");
+  loadVettingParticipation(json, ourDiv);
+  cldrSurvey.hideLoader();
+  cldrLoad.flipToOtherDiv(ourDiv);
+}
+
+function errorHandler(err) {
+  cldrRetry.handleDisconnect(err, json, "", "Loading forum participation data");
+}
+
+/**
+ * Get the AJAX URL to use for loading the Vetting Participation page
+ */
+function getAjaxUrl() {
+  const p = new URLSearchParams();
+  p.append("what", "vetting_participation");
+  p.append("s", cldrStatus.getSessionId());
+  return cldrAjax.makeUrl(p);
+}
+
+/**
+ * Populate the given div, given the json for Vetting Participation
+ *
+ * @param json
+ * @param ourDiv
+ */
+function loadVettingParticipation(json, ourDiv) {
+  nf = new Intl.NumberFormat();
+  const { missingLocalesForOrg, languagesNotInCLDR } = json;
+
+  // crunch the numbers
+  const { localeToData, totalCount, uidToUser } = calculateData(json);
+
+  ourDiv.id = "vettingParticipation";
+  const div = $(ourDiv);
+
+  // Front matter
+  div.append($("<h3>Locales and Vetting Participation</h3>"));
+  div.append(
+    $("<p/>", {
+      text: `Total votes: ${nf.format(totalCount || 0)}`,
+    })
+  );
+  if (missingLocalesForOrg) {
+    div.append(
+      $("<i/>", {
+        text: `“No Coverage” locales indicate that there are no regular vetters assigned in the “${missingLocalesForOrg}” organization.`,
+      })
+    );
+    if (languagesNotInCLDR) {
+      div.append(
+        $("<h4/>", {
+          text: "Locales not in CLDR",
+        })
+      );
+      div.append(
+        $("<i/>", {
+          text: `These locales are specified by Locales.txt for ${missingLocalesForOrg}, but do not exist in CLDR yet:`,
+        })
+      );
+      for (const loc of languagesNotInCLDR) {
+        div.append(
+          $("<tt/>", {
+            class: "fallback_code missingLoc",
+            // Note: can't use locmap to get a translation here, because locmap only
+            // has extant CLDR locales, and by definition 'loc' is not in CLDR yet.
+            text: `${loc}`, // raw code
+          })
+        );
+      }
+    }
+  }
+
+  // Chapter 1
```

Comment:
Chapter 1?

Reply:
I copied that along with the rest of the code in special/vetting_participation.js -- your guess is at least as good as mine what it means! If it means nothing to you, I'll delete it...

Maybe it means "here is the start of one of the first main sections of code in this function"? Often a comment in the middle of an excessively long function suggests the need for a subroutine; hopefully a better name could be invented than `chapterOne()`

Reply:
Aha, The original had one chapter and an epilogue.  I think it was just a section marker, and yes, it could be a subroutine.